content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -193,6 +194,11 @@ the following types of volumes:
 * Azure Disk
 * Portworx
 * FlexVolumes
+* CSI - {{< feature-state for_k8s_version="v1.14" state="alpha" >}}
```

Comment:
the tag is not showing up in the preview.  maybe CSI needs to be its own section, similar to what I did for topology

Reply:
you mean it is breaking into a new line right? hmm yeah.

Reply:
I have moved this to its own section.